tests/test_oxidation_state.py

```diff
@@ -23,3 +25,30 @@ def test_oxidation_state(cif_filename):
 
     results_dict = compute_oxidation_states(cif).get_dict()
     assert results_dict['prediction'] == OXIDATION_STATES[cif_filename]
+
+
+# def test_tag_atoms():
+#     """Test that atoms are correctly tagged and assigned charges."""
+#     from ase.io import read
+#     atoms = read(os.path.join(DATA_DIR, 'Cu-I-II-HKUST-1.cif'))
+#     atoms = tag_ase_atoms(atoms=atoms, oxi_results=OXIMACHINE_RUNNER.run_oximachine(atoms))
+#
+#     assert atoms[113].charge == -1  # Cu(I)
+#     assert atoms[113].tag == 1  # Cu(I)
+#     assert atoms[140].charge == -2  # Cu(II)
+#     assert atoms[140].tag == 2  # Cu(II)
+
+
+def test_cp2k_subsys():
+    """"Test the CP2K subsys features"""
+    from ase.io import read
+    atoms = read(os.path.join(DATA_DIR, 'Fe-MOF-74.cif'))
+    structure = StructureData(ase=atoms)
+
+    subsys = Cp2kSubsys(structure=structure,
+                        oxidation_states=OXIMACHINE_RUNNER.run_oximachine(atoms),
```

Comment:
Since it is taking some time, maybe print in the `report` that the oximachine is running

Reply:
thanks, done

Reply:
fixed